packages/main/src/plugin/extension-loader.ts

```diff
@@ -84,7 +84,7 @@ export class ExtensionLoader {
   private analyzedExtensions = new Map<string, AnalyzedExtension>();
   private watcherExtensions = new Map<string, containerDesktopAPI.FileSystemWatcher>();
   private reloadInProgressExtensions = new Map<string, boolean>();
-  private extensionState = new Map<string, string>();
+  protected extensionState = new Map<string, string>();
```

Comment:
not for this PR, but do we need to replace string by 'started' | 'failed' | 'stopped' ?

Reply:
Agree